modules/imgcodecs/test/test_gif.cpp

```diff
@@ -366,6 +366,54 @@ TEST(Imgcodecs_Gif, encode_IMREAD_GRAYSCALE) {
     EXPECT_EQ(decoded.channels(), 1);
 }
 
+// See https://github.com/opencv/opencv/issues/26924
+TEST(Imgcodecs_Gif, decode_disposal_method)
+{
+    const string root = cvtest::TS::ptr()->get_data_path();
+    const string filename = root + "gifsuite/disposalMethod.gif";
```

Comment:
Looks like it's a new file. Please create PR to opencv_extra repo with the same branch name.

Reply:
I'm sorry I created branch which has same name thit patch, but I forget to create PR. 

Please could you merge this pull request ? https://github.com/opencv/opencv_extra/pull/1239